.prettierignore

```diff
@@ -5,3 +5,4 @@
 /tmp/
 /vendor/bundle/
 /.rubocop_todo.yml
+outsavvy-sapho-events.html
```

Comment:
not sure this file should be here?

Reply:
This was a draft PR holdover when F wanted to see how far i had gotten